packages/sources/eth-balance/test/e2e/balance.test.ts

```diff
@@ -0,0 +1,91 @@
+import { assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+import { serverErrors, validationErrors } from '../../../../core/test-helpers/src/helpers'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  beforeAll(() => {
+    process.env.RPC_URL = process.env.RPC_URL || 'fake-rpc-url'
+  })
```

Comment:
For e2e tests we should not set this here. That makes it easier to know if the RPC_URL is set or not.

```suggestion
```